packages/client/src/__tests__/integration/happy/interactive-transactions-sqlite/test.ts

```diff
@@ -64,7 +54,7 @@ describe('interactive transaction', () => {
    *     Received promise resolved instead of rejected
    *     Resolved to value: [{"email": "user_1@website.com", "id": "0d258eae-1c22-4af1-8c95-68a17e995c2e", "name": null}]
    */
-  testIf(process.platform !== 'darwin' || !process.env.CI)('timeout override', async () => {
+  test('timeout override', async () => {
```

Comment:
If the reason this test was skipped is no longer an issue, the TODO comment above needs to be removed as well.